dns/dnscrypt-proxy/src/opnsense/mvc/app/models/OPNsense/Dnscryptproxy/General.xml

```diff
@@ -86,10 +86,10 @@
             <default>0</default>
             <Required>Y</Required>
         </tls_disable_session_tickets>
-        <fallback_resolver type="TextField">
+        <bootstrap_resolvers type="TextField">
```

Comment:
same here

Reply:
Why? Don't you want the UI config and its configuration to be consistent?